crates/fs/src/fs.rs

```diff
@@ -1519,6 +1519,100 @@ async fn file_id(path: impl AsRef<Path>) -> Result<u64> {
     .await
 }
 
+#[cfg(target_os = "windows")]
+async fn windows_create_symlink(target: PathBuf, path: &Path) -> Result<()> {
```

Comment:
does using `std::os::windows::fs::{symlink_file, symlink_dir}` not work for some reason?

Reply:
AFAIK, It does not work without developer mode

Reply:
Yes, creating symbolic links on Windows requires the corresponding permissions, either by enabling Developer Mode on Windows or by launching Zed as `Administrator`